First part of trials ready for review

joemast opened the pull request.

Commit:
Two first trials solved and ready for review

Commit:
Try to correct markdown

Commit:
One more try to correct markdown

Commit:
Fix markdown

ovaunit commented on src/Trials/BinarySum.php:
```diff
@@ -0,0 +1,18 @@
+
+namespace Trials;
+
+function binarySum (string $arg1, string $arg2): string
```
После имени функции не должно быть пробела (isPowerOfThree аналогично)

ovaunit commented on tests/BinarySumTests.php:
```diff
@@ -0,0 +1,10 @@
+
+namespace Tests;
+
+require_once "../src/Trials/BinarySum.php";
```
Если в строке нет переменных - нет смысла использовать двойные кавычки, лучше одинарные.
Аналогично в PowerOfThreeTests.php и ниже ("10", "01", ...)

ovaunit commented on src/Trials/PowerOfThree.php:
```diff
@@ -0,0 +1,26 @@
+
+namespace Trials;
+
+function isPowerOfThree (int $int): bool
```
Для -3, -27 функция возвращает true, но эти числа не являются степенью тройки.

joemast replied:
Это нечётные степени: -3 в 1 степени = -3, а -3 в 3 степени будет -27

Commit:
Make fixes on review comments
Make use of namespaces explicitly
Even more improvement for isPowerOfThree function - check -1 as a power of -3

Commit:
Make use of function from namespace

Commit:
Get rid of import statements

Commit:
Fibonacci numbers trial solved
